packages/fluentui/react-northstar/src/themes/teams/components/Carousel/carouselNavigationVariables.ts

```diff
@@ -52,6 +53,7 @@ export interface CarouselNavigationVariables {
 
 export default (siteVars: any): CarouselNavigationVariables => {
   return {
+    width: 300,
```

Comment:
Add it here to make it the same as for the `ItemsContainer`

Reply:
I think here we use values as rems, so you probably need to wrap the 300 with pxToRem

Reply:
Sure, I can wrap it.. Just set as it was for `ItemsContainer`.. So gonna change in both

Reply:
Actually I am using `pxToRem` here: 

```
transform: `translateX(${pxToRem(v.width / 2 - v.thumbnailWidth / 2 - +p.activeIndex * v.thumbnailWidth)})`,
```

Reply:
So going to leave as it is otherwise the calculation wouldn't work since v.width would be something like `"1.2234rem`

Reply:
Maybe @mnajdova knows better. :D

Reply:
Just make sure that this does not end up as px value in the styles. Otherwise it's ok if you need it for calculations to leave it as number.